acceptance-tests/Makefile.am

```diff
@@ -73,6 +73,9 @@ coreclr-runtest-managed: coreclr-validate test-runner.exe $(CORECLR_TESTSI_CS) $
 
 check-coreclr: coreclr-compile-tests coreclr-runtest-managed
 
+stresstest-coreclr: coreclr-validate ReliabilityFramework.exe $(CORECLR_STRESS_TESTS)
```

Comment:
I tried to follow a naming convention of `{test suite}-{target}` in this Makefile in order to have a clean separation between all the test suites (i.e. you can just type `coerclr-`, `roslyn-`, `whatever-` and hit tab on the shell to get all applicable targets).

Consider renaming it to `coerclr-stresstest`.